### ISSUE TITLE
[v2][auto-prompt] Change hotkey bindings

### ISSUE THREAD

**Issue by ajrosen** [aws/aws-cli#5715]
The key bindings for hiding/showing docs and help are inconvenient, especially on Macbooks.  F1, F2, and F3 are often used for common system functions.

I'd like to use Control or Alt/Command instead, borrowing from existing shells and editors.  Or provide a way to set the bindings in a configuration file.

**Comment by kdaily** [comment]
Thanks for the feature request, @ajrosen!

**Comment by tim-finnigan** [comment]
Checking in as this issue hasn't has any activity in a while. @ajrosen does the [PR](https://github.com/aws/aws-cli/pull/6818) linked above address what you were hoping with hotkey bindings? And @Damans227 did you see [the feedback](https://github.com/aws/aws-cli/pull/6818#issuecomment-1144165686) on your PR?

**Comment by ajrosen** [comment]
Yes, the PR looks exactly like what I was hoping for!  TBH I had completely forgotten about this, but I'm glad you/github did not.  I've subscribed to the PR.
